subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/release/TravisPlugin.java

```diff
@@ -74,5 +79,12 @@ public void execute(ReleaseNeededTask t) {
                 t.setPullRequest(isPullRequest);
             }
         });
+
+        GitCommitTask gitCommit = (GitCommitTask) project.getTasks().getByName(GitPlugin.GIT_COMMIT_TASK);
+        //can we construct the url from some Travis env variable?
+        //this needs to be safe for local invocation where there are no travis env variables.
+        String travisJobUrl = "https://travis-ci.org/mockito/shipkit/builds/288214072";
```

Comment:
I don't get it. Should it be project specific? PowerMock or Mockito should rather have some other Travis links.

I haven't check it, but couldn't [`TRAVIS_BUILD_ID`](https://docs.travis-ci.com/user/environment-variables/) be used here?

Reply:
>I haven't check it, but couldn't TRAVIS_BUILD_ID be used here?

Yes, that's the idea! We won't be hardcoding any url here!